crates/ironrdp-server/src/server.rs

```diff
@@ -44,13 +44,16 @@ pub struct RdpServerOptions {
 pub enum RdpServerSecurity {
     None,
     Tls(TlsAcceptor),
+    // for both hybrid + hybrid-ex
```

Comment:
suggestion: Could be part of the public documentation.
```suggestion
    /// Used for both hybrid + hybrid-ex
```